conductr_cli/sandbox_run_jvm.py

```diff
@@ -338,13 +308,12 @@ def extract_binary(path, binary_info):
         os.rmdir(extraction_subdir)
         return extraction_dir
 
-    core_binary_info = resolve_binary_info(ConductrComponent.CORE)
-    agent_binary_info = resolve_binary_info(ConductrComponent.AGENT)
+    core_info, agent_info = sandbox_common.resolve_conductr_info(image_dir)
 
     core_binary_path, agent_binary_path = resolve_binaries()
```

Comment:
Just a suggestion, feel free to skip if you feel this is unnecessary.

I think it will be much more readable if `core_info` and `agent_info` is passed into `resolve_binaries`, i.e.

```
     core_binary_path, agent_binary_path = resolve_binaries(core_info, agent_info)
```

Otherwise, the `resolve_binaries` as a backwardly declared reference to `core_info` and `agent_info`. Plus `core_info` and `agent_info` are passed into the `extract_binary` in the next two lines, so it's good to be consistent.